AttitudeManager/Src/AM.cpp

```diff
@@ -35,13 +35,93 @@ AttitudeManagerInput AttitudeManager::getControlInputs() {
     return temp;
 }
 
+AttitudeManager::AttitudeManager(Flightmode* control_algorithm):
+    control_algorithm_(control_algorithm),
+    motorInstances_(new MotorInstance_t[config::NUM_MOTORS]{})
+{
+    // Go through motor config list and count the # of motors of each axis type
+    uint8_t yawCount{0}, pitchCount{0}, rollCount{0}, throttleCount{0};
```

Comment:
I personally think slightly more verbose names might be helpful. Maybe "numYawMotors" or "numMotorsYaw". 

If our style guide is still relevant, I'd take a look at "Member Variables" section here: https://uwarg-docs.atlassian.net/wiki/spaces/ZP/pages/1827635203/Style+Guide

Reply:
I think this may be superseded by https://uwarg-docs.atlassian.net/l/cp/61ADnS0A

Reply:
Changed those variable to be more verbose, definitely agreed. I originally went with the style guide Aidan sent for my variables but not sure if there is something else you wanted me to improve that is not mentioned there